### PR TITLE
[PDI-19868] - Schedule perspective for the Administrator Role is different between PUC and PDI

### PR DESCRIPTION
**Pull request by andreramos89**
Added the method to obtain permissions from the server, and permission validation to the execution methods.

@smmribeiro 
@bcostahitachivantara 
@renato-s

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 56m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl scheduler
```
---
:ok_hand: All tests passed!
 
Tests run: **1745**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5423/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: scheduler/src/main/java/org/pentaho/platform/scheduler2/ws/ISchedulerService.java [hunk 1]
```diff
@@ -104,4 +104,6 @@ public String updateJobComplexTriggerWithJobName( String jobName, String jobId,
    */
   public int getSchedulerStatus() throws SchedulerException;
 
+  public boolean canStopScheduler();
```
Comment (smmribeiro) [inline scheduler/src/main/java/org/pentaho/platform/scheduler2/ws/ISchedulerService.java:107]: Adding a new method to an Interface may be "dangerous"...
We have to check this better...

Comment (andreramos89) [reply scheduler/src/main/java/org/pentaho/platform/scheduler2/ws/ISchedulerService.java]: Added default behaviour for interface mmethod

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 45m 16s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl scheduler
```
---
:ok_hand: All tests passed!
 
Tests run: **1745**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5423/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**